sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -10,11 +10,13 @@ import {
 import { AzureKeyCredential } from "@azure/core-auth";
 import { ClientOptions } from "@azure-rest/core-client";
 import createClient, { ModelClient } from "../../../src/index.js";
+import { DeploymentType } from "../types.js";
 
 const envSetupForPlayback: Record<string, string> = {
   AZURE_ENDPOINT: "https://endpoint.openai.azure.com/openai/deployments/gpt-4o/",
+  AZURE_EMBEDDINGS_ENDPOINT: "https://endpoint.openai.azure.com/openai/deployments/text-embedding-ada-002/",
   SUBSCRIPTION_ID: "azure_subscription_id",
-  AZURE_CLIENT_SECRET: "azureclientsecret"
+  AZURE_EMBEDDINGS_CLIENT_SECRET: "azureembeddingsclientsecret"
```

Comment:
I think API keys should just be called API keys, client secret is an Entra ID concept and using it in this context may cause confusion.